app/master/build.py

```diff
@@ -523,6 +527,22 @@ def _delete_temporary_build_artifact_files(self):
         end_time = time.time() - start_time
         self._logger.info('Completed deleting artifact files for {}, took {:.1f} seconds.', self._build_id, end_time)
 
+    def _update_prometheus_metrics(self):
+        serialized_time_val, estimation_error_val = 0, 0
+        for subjob in self._all_subjobs_by_id.values():
+            for atom in subjob.atoms:
+                if atom.actual_time is None:
+                    self._logger.warning('Could not load atom.actual_time for atom {} build {}', atom.command_string, self._build_id)
+                    return
+                if atom.expected_time is None:
+                    self._logger.warning('Could not load atom.expected_time for atom {} build {}', atom.command_string, self._build_id)
+                    return
+                serialized_time_val += atom.actual_time
+                estimation_error_val += abs(atom.actual_time - atom.expected_time)
+
+        serialized_build_time_seconds.observe(serialized_time_val)
```

Comment:
I agree, and maybe i'll do that in a follow up change, but request_latency was already a variable before I got here and it was lowercase. For consistency I want to keep mine lowercase as well.  If request_latency was only used in a couple places then i'd just change it here, but it's used in 40 places. I don't want to clutter up my diff with 40 lines that don't belong in it.